management/azure_mgmt_confluent/lib/profiles/latest/confluent_latest_profile_client.rb

```diff
@@ -2,15 +2,15 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 
-require 'profiles/latest/compute_module_definition'
-require 'profiles/latest/modules/compute_profile_module'
+require 'profiles/latest/confluent_module_definition'
+require 'profiles/latest/modules/confluent_profile_module'
 
-module Azure::Compute::Profiles::Latest
+module Azure::Confluent::Profiles::Latest
```

Comment:
@kairu-ms was deleting this from `azure_mgmt_compute` intentional?  This appears to have made it into a released version of `azure_mgmt_compute` preventing it from being required.

https://github.com/Azure/azure-sdk-for-ruby/issues/2818

Reply:
Thanks for your report. I will fix it soon.

Reply:
Unsure of your build/release pipeline process but I don't understand why you wouldn't fix it before merging this :shrug:

Reply:
Sorry for that issue. It has been fixed in version 0.21.1. Please try again.